### PR TITLE
Add tests for explicit model control in Python backend

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by CoderHam — CHANGES_REQUESTED** [review]
Can you change the test name from explicit to model_control / explicit_model_control?

**Comment by tanmayv25** [comment]
@Tabrizian Should we add these tests in `lifecycle`? We have explicit model load/reload for other platforms in L0_lifecycle.

File: qa/L0_backend_python/explicit/explicit_model_test.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            model_name=model_name,
+            inputs=inputs,
+            outputs=[httpclient.InferRequestedOutput('OUTPUT0')])
+        output_numpy = result.as_numpy('OUTPUT0')
+        self.assertTrue(np.all(input0_data == output_numpy))
+
+    def test_model_reload(self):
+        model_name = "identity_fp32"
+        with httpclient.InferenceServerClient("localhost:8000") as client:
+            for _ in range(5):
```
Comment (tanmayv25) [inline qa/L0_backend_python/explicit/explicit_model_test.py:56]: You can check the `model_name` is not already loaded in the server before proceeding.

**Comment by Tabrizian** [comment]
@tanmayv25 I looked into L0_lifecycle. It looks like a good place to add Python backend but the tests need some changes to make them compatible with other backends so it can take some time. I created a ticket for it.

**Review by tanmayv25 — APPROVED** [review]

**Review by CoderHam — APPROVED** [review]

**Pull request merged**